src/tracker/encoder/os_net/osnet.py

```diff
@@ -239,6 +240,10 @@ def __init__(self, in_channels, out_channels, reduction=4, T=4, **kwargs):
         self.downsample = None
         if in_channels != out_channels:
             self.downsample = Conv1x1Linear(in_channels, out_channels)
+        if normalize_instance:
```

Comment:
This doesn't sound like a boolean. Perhaps `should_normalize_instance`? Perhaps that's too long. Any other ideas?

Reply:
maybe `should_IN`? I'm still uncertain about some of these acronyms.